.github/actions/build-windows-qt/action.yml

```diff
@@ -51,6 +51,17 @@ runs:
       shell: pwsh
       run: ./Build-Dependencies.ps1 -Dependencies "Qt${{ inputs.qtVersion }}" -SkipBuild -SkipUnpack -Target ${{ inputs.target }} -Configuration ${{ inputs.config }} -Shared
 
+    - name: Install Qt
```

Comment:
Don't use third party actions, if you need a host-version of Qt, wait for the associated x64 job to finish, then use that artifact.

Reply:
okay.

Reply:
Let us know if that is not feasible as-is - in doubt we can and should amend the current Qt build workflow to generate what this workflow needs. Also I don't think I have seen associated steps in the build scripts ensuring that a host installation of Qt is available to cross compile it?

Reply:
I think that's possible in theory. I will need to dig a little bit about the CI though.